aten/src/ATen/Device.h

```diff
@@ -1,2 +1,2 @@
 #pragma once
-#include <ATen/core/Device.h>
+#include <c10/Device.h>
```

Comment:
Quick question: should we put it to c10/core?

Reply:
I am on the record for wanting user-visible classes to live at the top-level!

Reply:
Sounds good to me ;)